src/scss/colors.scss

```
@@ -0,0 +1,38 @@
+@use "sass:color";
+
+$colors-enum: (
+  primary: #7c46fb,
+);
+$shades-enum: (5, 10, 14, 20, 50);
+$alphas-enum: (0.18, 0.2, 0.24, 0.35, 0.5, 0.56, 0.65, 0.8);
+
+@function hexToRGB($hex) {
+  @return red($hex), green($hex), blue($hex);
+}
+
+:root {
+  --primary-color-deprecated: #f6e652;
+
+  --primary-content--under: #000000;
+  --primary-content--under-rgb: #{hexToRGB(#000000)};
+  --primary-content--over: #ffffff;
+  --primary-content--over-rgb: #{hexToRGB(#ffffff)};
+  --primary-accent--over: #ffffff;
+
+  @each $colorName, $color in $colors-enum {
```

Snake case in (S)CSS, but here just `$name` should suffice.

```suggestion
  @each $name, $color in $colors-enum {
```

---

fixed in 8e20515ab04bf4df178d893dadb185d0e7b6126e